pkg/convert/pot_test.go

```diff
@@ -85,6 +85,94 @@ msgstr ""
 #, fuzzy
 msgid "Hello, world!"
 msgstr "Bonjour le monde!"
+`),
+		},
+		{
+			name: "msgstr with curly braces inside",
+			input: model.Messages{
+				Language: language.English,
+				Messages: []model.Message{
+					{
+						ID:          "Hello, world!",
+						Message:     `{Bonjour \{\} le monde!}`,
+						Description: "A simple greeting",
+						Status:      model.MessageStatusFuzzy,
+					},
+				},
+			},
+			expected: []byte(`msgid ""
+msgstr ""
+"Language: en\n"
+#. A simple greeting
+#, fuzzy
+msgid "Hello, world!"
+msgstr "Bonjour {} le monde!"
+`),
```

Comment:
Can we reduce all test cases to what's really under the test? Fuzzy and description add noise.

```suggestion
				Messages: []model.Message{
					{
						ID:          "Hello, world!",
						Message:     `{Bonjour \{\} le monde!}`,
					},
				},
			},
			expected: []byte(`msgid ""
msgstr ""
"Language: en\n"
msgid "Hello, world!"
msgstr "Bonjour {} le monde!"
`),
```